### ISSUE TITLE
Apple Home App is showing devices with "no answer", Eve App all perfect

### ISSUE THREAD

**Issue by DJay79** [nfarina/homebridge#1325]
Since yesterday and out of the blue, Apple Home App is showing all homebridge devices as "no answer". At first start it is refreshing the devices but after a second they show up as not available.
Strange when I push one of the devices without any complain it is then showing the acutual status. Pushing it again it also works like expected i.e. Fan on/off. All other devices are still marked as not available until I push the affected device.
On the other hand in the Elgato Eve App everything works normal and is accessible immediately.
I reset HomeKit in the Home App by removing the Home completely, cose the App and restart my iPhone. Then add homebridge again in Home App but the same issue than before.
Buy the way Siri and control center works as expected and even show correct the actual state of the devices. It's online inside the Home App.
Any suggestions and how to fix this?
Thanks in advance for you help and reply.

homebridge@0.4.20 | n@2.1.7 |node@8.0.0 | npm@5.0.2

**Comment by walksonair** [comment]
Yes, same issue with me but my eCobee is the one that is still working. All homebridge stuff stopped working. When I run Debug= * homebridge, the last line is _[6/5/2017, 3:32:40 PM] [SmartThings] SmartThings Hub Communication Established_

**Comment by walksonair** [comment]
Nevermind, I deleted the PERSIST and ACCESSORIES folders and then removed my homebridge bridge from the Home app. Re-added the bridge and everything is working peachy again. Wish I knew if there was a better way than this nuke method.

**Comment by DJay79** [comment]
@walksonair Thanks for your reply. I also did remove persist and accessories, removed my home completely from home app and start from scratch. A bit work, but this fixed it.

**Comment by DJay79** [comment]
I unfortunately have to reopen this. No Idea what causes the problem. Everything works great in the Eve App. Homebridge is running the whole day perfect.
But Apple Home App shows the devices with "No answer" until I push the button of a single device. Then this specific device shows up correct. All others devices, even when the the Home App reloads, did not. But I can repeat pressing every single at once and the state is correct. What is strange, that devices show up correct in the control center and also Siri is able to control them as expected.
Heres a [YouTune Video](https://youtu.be/jzEP4yjCPM4) to show what the situation is in Home App and Eve App.
Thanks in advance for hopefully a non nuke solution that does not mean to remove homebridge again.

**Comment by walksonair** [comment]
Well, I'm at the same point I was before nuking. Thinking I may have to do that again just to get things working decently again. I have run in DEBUG and looks like the connection to HomeBridge is not happening anymore...anyone care to help further debug?

**Comment by walksonair** [comment]
Well I nuked it...and I mean I really nuked it this time. I first tried updating everything..including my ST app. Then I tried removing the folders persist and accessories. Nada. Then I just removed my entire HOME because my iPad was showing something different from my iPhone. 

Well, I added everything and its setup once again. The only thing now is a discrepancy between where my iPad has each accessory to a room and the iPhone shows DEFAULT ROOM for some accessories. Weird.
